### ISSUE TITLE
Conker's Bad Fur Day (U) freezes

### ISSUE THREAD

**Issue by MASF90s** [loganmc10/m64p#379]
**Crashes:**
Conker's Bad Fur Day freezes around to 6-7 minutes of gameplay. I can't pinpoint exactly what causes it, but I am positive it happens between 6 mins and 12 mins on average if I get lucky enough for it not to crash.

**Specs:**
 - OS: Windows
 - Version: 10
 - GPU: RTX 3090
 - CPU: Intel Core i9 12900K
 - RAM: 64 GBs

**Comment by loganmc10** [comment]
Does it crash (the emulator closes) or does the game freeze?

**Comment by MASF90s** [comment]
The emulator stays open, the game itself freezes.

**Comment by MASF90s** [comment]
I've tested the game in the recent version **v2023.07.4** and can confirm the game is still freezing. Also, let me know if reporting the freezing is helpful or not.

**Comment by loganmc10** [comment]
Still after 6-7 minutes?

**Comment by MASF90s** [comment]
Yeah pretty much, I went to try It's War in Multiplayer and it crashed around 6-7 minutes. The previous freezes was through story.

**Comment by MASF90s** [comment]
I played It's War in Multiplayer and I ran into the sniper room and the game froze. I think the freezing might have happened sooner than it did last time. I wasn't paying close attention but it felt that way. This time was in 3-4 unless I am remembering wrong.

**Comment by MASF90s** [comment]
In my testing in **v2023.08.4** Conker freezes at around 12-17 minutes. I was able to play much longer but it will freeze eventually.

**Comment by AzureNova** [comment]
Can confirm this happens to me too. There isn't an exact time or activity that causes it. It happened to me several times in a 3 hour period, captured on this stream, if that helps at all.
Time stamps for crashes are: 37:21, 1:54:19, 2:38:15
https://rumble.com/v53rrfl-conkering-our...-addiction-conkers-bad-fur-day.html

**Comment by petran79** [comment]
In my case it froze right after being resurrected and brought back to stage. I made a save state if it helps
[Conker2.zip](https://github.com/user-attachments/files/18028776/Conker2.zip)

**Comment by Jj0YzL5nvJ** [comment]
Cached interpreter issue? #267

**Comment by petran79** [comment]
> Cached interpreter issue? #267

I am not sure. How do I enable or disable cached and static?

**Comment by smoothstill** [comment]
I am having the same issue, Conker's Bad Fur Day (U) freezes during multiplayer within the first 10 minutes. No clear pattern to it.
